src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useMemo } from 'react';
+import { createAvatar } from '@dicebear/core';
+import { initials } from '@dicebear/collection';
+
+interface InterfaceAvatarProps {
+  name: string;
+  alt?: string;
+  size?: number;
+}
+
+const Avatar = ({ name, alt, size }: InterfaceAvatarProps): JSX.Element => {
+  const avatar = useMemo(() => {
+    return createAvatar(initials, {
+      size: size || 128,
+      seed: name,
+    }).toDataUriSync();
+  }, [name]);
+
+  const svg = avatar.toString();
+
+  return <img src={svg} alt={alt} />;
```

Comment:
@akhilender-bongirwar you need to handle the case where, alt will be undefined, also add a style parameter of string over here like I have done in `ChangeLanguageDropdown.tsx` component, this would give other devs the flexibility to change the styling of the avatar as they would need.

Reply:
I've addressed the undefined `alt` value by providing a default that can be overridden. Additionally, I've implemented custom handling of the `className` through the `avatarStyle` prop, granting developers flexibility in styling the avatar to their specific needs. I intentionally omitted a default style, as the existing behavior meets expectations without it.